src/main/java/me/creighton/encodedid/IAlphabet.java

```diff
@@ -14,12 +14,15 @@
 
 public interface IAlphabet {
 
-  String LOWER_ALPHA = "abcdefghijklmnopqrstuvwxyz";
-  String UPPER_ALPHA = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
-  String ALPHA = LOWER_ALPHA + UPPER_ALPHA;
-  String DIGIT = "0123456789";
-  String SPECIAL = ":._~!;"; // '-' is excluded as default separator
-  String LEGAL_URI_CHARACTER_SET = ALPHA + DIGIT + SPECIAL;
+  public final String LOWER_ALPHA = "abcdefghijklmnopqrstuvwxyz";
```

Comment:
Minor feedback - `public` and `final` keywords for constants on an interface are implied, but it syntactically correct. IntelliJ claims they're redundant.

Reply:
I'm moving all constants to a constants class. This mirrors what was already done in OTPConstants.